src/core/reducers/autocomplete.js

```diff
@@ -0,0 +1,50 @@
+export const AUTOCOMPLETE_LOADED = 'core/autocomplete/AUTOCOMPLETE_LOADED';
+export const AUTOCOMPLETE_STARTED = 'core/autocomplete/AUTOCOMPLETE_STARTED';
+
+const initialState = {
+  suggestions: [],
+};
+
+export function autocompleteStart({ errorHandlerId, query }) {
+  if (!errorHandlerId) {
+    throw new Error('errorHandlerId is required');
+  }
+
+  if (!query) {
+    throw new Error('query is required');
+  }
+
+  return {
+    type: AUTOCOMPLETE_STARTED,
+    payload: { errorHandlerId, query },
+  };
+}
+
+export function autocompleteLoad({ results }) {
+  if (!results) {
+    throw new Error('results is required');
+  }
+
+  return {
+    type: AUTOCOMPLETE_LOADED,
+    payload: { results },
+  };
+}
+
+export default function reducer(state = initialState, action) {
+  const { payload } = action;
+  switch (action.type) {
+    case AUTOCOMPLETE_STARTED:
+      return state;
+    case AUTOCOMPLETE_LOADED:
+      return {
+        ...state,
+        suggestions: payload.results
+          // TODO: cf. https://github.com/mozilla/addons-server/issues/6189
```

Comment:
You got it right! 😃 

More seriously, sorry for that, it was not clear at all.